### PR TITLE
chore(JavaScript): add stack and queue implementation

### PR DESCRIPTION
**Pull request by underager**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [ ] My submission has a proper and user-friendly description of the algorithm
- [ ] My submission has the time complexity of the algorithm
- [ ] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**
Following algorithm has been implemented in Javascript
1. Stack Implementation
2. Queue Implementation
3. Stack using two queue implementation

**Other information:**
Reference Issue : #638

### CODE REVIEW

File: algorithms/JavaScript/README.md [hunk 1]
```diff
@@ -22,3 +22,10 @@
 ## Strings
 - [Palindrome](src/strings/palindrome.js)
 - [Sequence](src/strings/sequence.js)
+
+## Stack
+- [Stack Implementation](src/stack/stackImpl.js)
+- [Stack Using Two Queue](src/stack/usingTwoQueue.js)
+
+## Queue
+- [Queue Implementation](src/Queue/queueImpl.js)
```
Comment (ming-tsai) [inline algorithms/JavaScript/README.md:31]: Hi @underager,
Please rename the folder name to plural and lowercase and the file name as a suggestion.
```suggestion
## Stack
- [Stack Implementation](src/stacks/stack.js)
- [Stack Using Two Queue](src/stacks/two-stack.js)

## Queue
- [Queue Implementation](src/queues/queue.js)
```

**Comment by ming-tsai** [comment]
Hi @underager,
Are you finished modifying?

**Comment by underager** [comment]
> 

Yes I have renamed the directory and file as suggested and the same is reflected in the latest commit, please let me know if there is anything else, thank you.

File: algorithms/JavaScript/src/queue/queueImpl.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+class Node {
```
Comment (ming-tsai) [inline algorithms/JavaScript/src/queue/queueImpl.js:1]: Please delete this file

File: algorithms/JavaScript/src/stack/stackImpl.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+class Stack {
```
Comment (ming-tsai) [inline algorithms/JavaScript/src/stack/stackImpl.js:1]: Please delete this file

File: algorithms/JavaScript/src/stack/usingTwoQueue.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+// Implement Stack using two queues
```
Comment (ming-tsai) [inline algorithms/JavaScript/src/stack/usingTwoQueue.js:1]: Please delete this file

File: algorithms/JavaScript/README.md (hunk 1 above)
Comment (ming-tsai) [reply algorithms/JavaScript/README.md:31]: Update the reference @underager

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @underager,
Please delete these file

File: algorithms/JavaScript/src/queue/queueImpl.js (hunk 2 above)
Comment (underager) [reply algorithms/JavaScript/src/queue/queueImpl.js]: This file has been deleted.

File: algorithms/JavaScript/src/stack/stackImpl.js (hunk 3 above)
Comment (underager) [reply algorithms/JavaScript/src/stack/stackImpl.js]: This file has been deleted.

File: algorithms/JavaScript/src/stack/usingTwoQueue.js (hunk 4 above)
Comment (underager) [reply algorithms/JavaScript/src/stack/usingTwoQueue.js]: This file has been deleted

**Comment by underager** [comment]
I hope the latest commits take care of everything. Sorry for inconvenience.

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**